src/manipulation.js

```diff
@@ -26,7 +26,7 @@ define( [
 	dataPriv, dataUser, acceptData ) {
 
 var
-	rxhtmlTag = /<(?!area|br|col|embed|hr|img|input|link|meta|param)(([\w:-]+)[^>]*)\/>/gi,
+	rxhtmlTag = /<(?!area|br|col|embed|hr|img|input|link|meta|param)(([a-z][^\x20\t\r\n\f\/\0>]*)[^>]*)\/>/gi,
```

Comment:
Without the two negated `>`, some tests fail such as [this one](https://github.com/jquery/jquery/blob/master/test/unit/manipulation.js#L1619-L1621). It seems that before the `\w` was taking care of the job of the first negation, but now we're finding the name-terminating characters explicitly, we'll need to take care of this ourselves.